src/builder/BridgeRoutes.sol

```diff
@@ -196,4 +197,18 @@ library Across {
             )
         );
     }
+
+    // Returns whether or not an asset is bridged non-deterministically. This applies to WETH/ETH, where Across will send either ETH or WETH
+    // to the target address depending on if address is an EOA or contract.
+    function isNonDeterministicBridgeAction(HashMap.Map memory assetsBridged, string memory assetSymbol)
+        internal
+        pure
+        returns (bool)
+    {
+        uint256 bridgedAmount = HashMap.contains(assetsBridged, abi.encode(assetSymbol))
+            ? HashMap.getUint256(assetsBridged, abi.encode(assetSymbol))
+            : 0;
+        return bridgedAmount != 0
```

Comment:
Weirdly also feels like `> 0` is a tad more natural for the intent here, though logically the same result

Reply:
Yeah I agree, I can change it

Reply:
Done